app/pages/angelfish.tsx

```diff
@@ -30,14 +30,14 @@ const Home: React.FC<HomePageProps> = (props: HomePageProps) => {
         <meta name="robots" content="noindex, nofollow" />
         <meta name="googlebot" content="noindex, nofollow" />
       </Head>
-      <section className="pb-12 pt-12 md:pt-16">
+      <section className="margin-top-2xl margin-bottom-xl md:pt-16">
```

Comment:
I'll do the rest of these in a seperate PR

Reply:
I think it's mostly done in my PR but will need the design eye ;)